cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -69,12 +69,21 @@ def self.config(env)
             proxy: 'dashboard',
             headers: LANGUAGE_HEADER,
             cookies: whitelisted_cookies
+          },
+          {
+            path: %w(
+              /
+              /learn*
+              /congrats
+            ),
+            headers: LANGUAGE_HEADER,
+            cookies: LANGUAGE_COOKIE
           }
         ],
-        # Default Pegasus paths are cached but language-specific, whitelist only language cookie/header.
+        # Remaining Pegasus paths are English-only and don't require any extra headers or cookies.
```

Comment:
We should make sure there is really clear documentation somewhat that explains how to update the non-english-only whitelist, and make sure we verbally explain this requirement to the entire dev team. I definitely didn't know about that.